# RFID标签数组
std_msgs/Header header
RfidTag[] tags          # 检测到的标签列表
uint32 total_tags       # 总标签数量
uint32 total_reads      # 总读取次数